cdc/entry/mounter.go

```diff
@@ -456,10 +468,14 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *model.TableInfo, row *rowKVEntr
 
 	schemaName := tableInfo.TableName.Schema
 	tableName := tableInfo.TableName.Table
+	var intRowID int64
+	if row.RecordID.IsInt() {
+		intRowID = row.RecordID.IntValue()
+	}
 	return &model.RowChangedEvent{
 		StartTs:          row.StartTs,
 		CommitTs:         row.CRTs,
-		RowID:            row.RecordID,
+		RowID:            intRowID,
```

Comment:
The `RowID` is not accurate if it is a common handle. Maybe we should mark this field as deprecated. (This field is also not added in the document)